### ISSUE TITLE
Old practices

### ISSUE THREAD

**Issue by maciejmyslinski** [radubrehar/react-examples#2]
Check [jsx-loader github page](https://github.com/petehunt/jsx-loader).
They say you should use **babel-loader**

**Comment by radubrehar** [comment]
Indeed, this has gotten a outdated. I'm using babel-loader for a few months now, but have forgotten to update this repo.

**Comment by Download** [comment]
I just read your tutorial and liked it. Any idea whether you will update it and make a follow-up?

**Comment by cimm** [comment]
Indeed, great tutorial. Was a bit disappointed at the end to find out it's kinda old and it's no longer how things are done. I found https://www.twilio.com/blog/2015/08/setting-up-react-for-es6-with-webpack-and-babel-2.html which looks like a similar but more modern version of this tutorial.
